src/platform/Kconfig

```diff
@@ -374,6 +390,20 @@ config CAVS_VERSION_2_5
 	help
 	  Select for CAVS version 2.5
 
+config ACE
+	bool
+	default n
+	select INTEL
+	select XT_INTERRUPT_LEVEL_2
+	select XT_INTERRUPT_LEVEL_5
+	select SCHEDULE_DMA_SINGLE_CHANNEL
```

Comment:
you don't even initialise the DMA scheduling domain. Is this needed? Maybe we should modify SOF to not require this. Can be done separately of course